.github/workflows/docs-release.yaml

```diff
@@ -0,0 +1,115 @@
+name: "Release Docs: package the docs and publish to Docs the repo"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      version:
+        required: true
+        type: string
+
+concurrency:
+  # Note that people may spam the post-commit pipeline on their branch, and
+  # we have this docs pipeline in the post-commit pipeline, then people
+  # would have to wait until the previous one fully completes. That may be
+  # ok because each post-commit pipeline definitely takes more than 30 min
+  group: "pages-${{ github.ref }}"
+  cancel-in-progress: false
+
+jobs:
+  package-push-docs:
+    strategy:
+      # Do not fail-fast because we need to ensure all tests go to completion
+      # so we try not to get hanging machines
+      fail-fast: false
+      matrix:
+        arch: [grayskull]
+    env:
+      DOCS_VERSION: ${{ inputs.version }}
+      DOCS_REPO: tenstorrent.github.io
+      ARCH_NAME: ${{ matrix.arch }}
```

Comment:
`ARCH_NAME` and `LD_LIBRARY_PATH` may actually be unnecessary here now.

Reply:
👍 The are both present.